dm/common/common.go

```diff
@@ -65,7 +65,9 @@ var (
 	// StageValidatorKeyAdapter is used to store the running stage of the validator.
 	// k/v: Encode(source-id, task-name) -> the running stage of the validator.
 	StageValidatorKeyAdapter KeyAdapter = keyHexEncoderDecoder("/dm-master/stage/validator/")
-
+	// LightningCoordinationKeyAdapter is used to store the coordination information of lightning.
+	// k/v: Encode(task-name, source-id) -> lightning status.
+	LightningCoordinationKeyAdapter KeyAdapter = keyHexEncoderDecoder("/dm-master/lightning/")
```

Comment:
Is `"/dm-master/lightning/"` too general? How about `"/dm-master/lightning-coord/"` or ``"/dm-master/lightning-status/"`